yb-voyager/cmd/exportData.go

```diff
@@ -277,6 +295,7 @@ func exportData() bool {
 	//finalTableList is with leaf partitions and root tables after this in the whole export flow to make all the catalog queries work fine
 
 	if changeStreamingIsEnabled(exportType) || useDebezium {
+		exportPhase = dbzm.MODE_SNAPSHOT
```

Comment:
should we set this even for offline migration?

Reply:
Yes it is set for all the cases.